initial/src/main/java/com/smartgun/model/incident/Incident.java

```diff
@@ -41,4 +42,12 @@ public int getEndTime() {
     public IncidentType getIncidentType() {
         return incidentType;
     }
+
+    public void setIncidentType(IncidentType incidentType) {
+        this.incidentType = incidentType;
+    }
+
+    public int getDurationTime() {
+        return durationTime;
+    }
```

Comment:
skoro zwracany jest durationTime, jest też przeznaczona do tego zmienna, to czemu nie ma przypisania do niej?